sources/packages/web/src/views/institution/locations/request-a-change/request-a-change.vue/AvailableToChange.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <available-to-change-summary :location-id="locationId" />
```

Comment:
Is `available-to-change-summary` created as a component with the intention to be shared?
If it will not be shared or its creation will not bring better code readability, should we have a component for such cases?
Same for the similar ones in the PR.

Reply:
we don't have plans to share I believe. I just kept it as a component, but I can revert, as it is the only logic inside the view/

Reply:
I saw the change. IMO it looks better @ann-aot.